test/pg-cdc/pg-source-errors.td

```diff
@@ -0,0 +1,43 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
sorry for the final nit, can you rename this file to `empty-publication.td`? thanks!

Reply:
sure!